styles/arch-user-repository/catppuccin.user.css

```
@@ -0,0 +1,310 @@
+/* ==UserStyle==
+@name Arch user repository Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/arch-user-repository
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/arch-user-repository
+@version 0.9.0
```

we don't need to change this until it reaches main
```suggestion
@version 0.1.0
```